src/engraving/style/styledef.cpp

```diff
@@ -993,13 +993,13 @@ const std::array<StyleDef::StyleValue, size_t(Sid::STYLES)> StyleDef::styleValue
     { Sid::staffTextFrameBgColor,         "staffFrameBgColor",            PropertyValue::fromValue(draw::Color::transparent) },
 
     { Sid::rehearsalMarkFontFace,         "rehearsalMarkFontFace",        "Edwin" },
-    { Sid::rehearsalMarkFontSize,         "rehearsalMarkFontSize",        14.0 },
+    { Sid::rehearsalMarkFontSize,         "rehearsalMarkFontSize",        8.0 },
     { Sid::rehearsalMarkLineSpacing,      "rehearsalMarkLineSpacing",     1.0 },
     { Sid::rehearsalMarkFontSpatiumDependent, "rehearsalMarkFontSpatiumDependent", true },
     { Sid::rehearsalMarkFontStyle,        "rehearsalMarkFontStyle",       int(FontStyle::Bold) },
     { Sid::rehearsalMarkColor,            "rehearsalMarkColor",           PropertyValue::fromValue(draw::Color::black) },
-    { Sid::rehearsalMarkAlign,            "rehearsalMarkAlign",           Align(AlignH::HCENTER, AlignV::BASELINE) },
-    { Sid::rehearsalMarkFrameType,        "rehearsalMarkFrameType",       int(FrameType::SQUARE) },
+    { Sid::rehearsalMarkAlign,            "rehearsalMarkAlign",           Align(AlignH::LEFT, AlignV::BASELINE) },
+    { Sid::rehearsalMarkFrameType,        "rehearsalMarkFrameType",       int(FrameType::NO_FRAME) },
```

Comment:
Why no frame for rehearsal marks anymore?

Reply:
@Jojo-Schmitz there was no frame in "old version" of gp-import, and as I'm fixing new import frame appeared.
I changed it mostly to raise the question. Anyway now I see it's all wrong =)